agentops/client.py

```diff
@@ -43,7 +43,7 @@ class Client(metaclass=MetaClient):
         max_queue_size (int, optional): The maximum size of the event queue. Defaults to 100.
         tags (List[str], optional): Tags for the sessions that can be used for grouping or 
             sorting later (e.g. ["GPT-4"]).
-        override (bool): Whether to override and LLM calls to emit as events.
+        override_llm_calls (bool): Whether to override LLM calls and emit LLMEvents.
```

Comment:
intrument here as well. Don't forget to change the text of the message to also say "Whether to instrument LLM calls and emit LLMEvents."